README.md

```diff
@@ -1,2 +1,73 @@
 # tk-react-exercise
 Onboarding React exercise 
+
```

Comment:
Please always include some context or instructions in the PR. I understand that the create-react-app boilerplate was used but I wasn't sure where and how to run the backend side of your application.

Also, it helps to understand some decisions you made while developing the project so we can discuss them in more detail

Reply:
Aha of course! I'll add that bit of info in to the readme as it's not testable/usable without running the server-side 😄